include/hw/virtio/virtio-dmabuf.h

```diff
@@ -18,11 +18,41 @@
 #include <glib.h>
 #include "qemu/uuid.h"
 
-
+/**
+ * virtio_add_dmabuf() - Add a new dma-buf resource to the table
```

Comment:
This is the code style to have function names in the description?

Reply:
I do not think there are general guidelines for this. It is similar to how other headers are documented in the same folder.